### ISSUE TITLE
Adding romhacks

### ISSUE THREAD

**Issue by noradninja** [Rinnegatamante/DaedalusX64-vitaGL#64]
I have an english patched version of Animal Forest- do I need to add it to roms.ini for it to work correctly? Right now it loads and I can hear audio but there are no graphics on screen.

**Comment by stromdenver** [comment]
Was it a Japanese rom with an English patch?

**Comment by noradninja** [comment]
Yes.

**Comment by stromdenver** [comment]
the patch changes the language but the rom region is still japan so I don't see why you have to annotate it in Roms.ini

**Comment by noradninja** [comment]
Yeah I just wasn't sure if there was a CRC check of some kind. Just seems odd that applying a patch to edit the language would make a previously working game no longer display.

**Comment by stromdenver** [comment]
The same thing happened with rugrats scavenger hunt was heard but could not be seen waiting for an update of Nightly.
now if you see the treasure hunt rugrats

**Comment by stromdenver** [comment]
sorry my english is not my primary language

**Comment by noradninja** [comment]
I think I get you. I'll keep the patched rom around and see if later updates correct the issue.

**Comment by stromdenver** [comment]
where to download the rom for me
to try it and I inform you

**Comment by noradninja** [comment]
[https://mega.nz/file/ZRQV2Ipb#f1Qn_DXCArPaK0HI7JCgO7ThH1SynNt5C3t2oCbLO3s](url)

**Comment by stromdenver** [comment]
I have tested the game with patches and without patches in all daedalus x64 builds and it does not open the game in any of them

**Comment by stromdenver** [comment]
what day did you try it the first time

**Comment by noradninja** [comment]
I tested an unpatched version two day ago (worked fine although frame rate was about half and there were menu drawing issues), tested this patched one today. I update to the latest nightly each day. I have loaded this rom with mupen64 Plus and it works there, which leads me to believe it is an issue with Daedalus.

![Screen Shot 2020-05-13 at 1 23 46 PM](https://user-images.githubusercontent.com/6692990/81844506-312f6000-951d-11ea-8ab6-a80dec1c7994.png)
